doc/docs/concepts.md

```diff
@@ -0,0 +1,123 @@
+
+### Extraction
+
+FHIR Data Pipes is designed to work with FHIR source data in various forms: 
+
+* Valid FHIR Server API (e.g. HAPI FHIR)
+* FHIR facade implementations (e.g. OpenMRSv3)
+* Bulk Export API data
+* ndjson representation
+
+### Transformation
+
+FHIR Resources are transformed into a "Parquet on FHIR" format:
+
+* Uses a forked version of [Bunsen library](https://github.com/google/fhir-data-pipes/tree/master/bunsen) to transform from FHIR (_current support for STU3, R4_) to the SQL-on-FHIR-v1 schema. 
+* Configurable support for FHIR profiles and extensions
+* (Optional) In-pipeline 'flattening' of FHIR data using [ViewDefinition](https://build.fhir.org/ig/FHIR/sql-on-fhir-v2/StructureDefinition-ViewDefinition.html) resources - [read more](#viewdefinition-resource) 
+
+### Loading
+
+FHIR Data Pipes supports different SQL Data Warehouse options depending on the needs of the project. These include:
+
+* Loading Parquet into an OLAP data warehouse such as SparkSQL (which can be deployed on a single machine or as a distributed query engine) or DuckDB (single machine in-process)
+* Traditional Relational data warehouse such as PostgreSQL (when using the FHIR ViewDefinitions resources to generate materialized views)
+
+## Pipeline Controller Module
+A user-interface wrapper for the FHIR Data Pipes Pipelines, integrating "full", "incremental", and "merger" pipelines together. 
+
+* The Pipelines Controller is built on top of pipelines and shares many of the same settings
+* Using the controller module you can schedule periodic incremental updates or use a [web control panel](link_to_section_in_adv_guide) to start the pipeline manually
+
+## Flexible and scalable deployment approaches
+
+There are a number of different deployment approaches - see table below. 
+
+Choosing the right approach, comes down to evaluation of a number of factors including the specific use case, requirements of the project, future scalability needs and capabilities of the team. 
+
+| Scenario | Approach | Considerations |
+| -------- | ----------| -------------- |
+|  Simple relational database to power dashboards or reporting | Custom schema defined as ViewDefinition Resources - [see tutorial](tutorial_lossy_example/) | By design this will provide a constrained set of variables in the views |
+| Need for a horizontally scalable architecture | Parquet based DWH and distributed query engine (e.g. SparkSQL) - [see tutorial](tutorial_add_dashboard/) | Will need to manage both distributed storage (Parquet files) and a distributed query engine |
+| Parquet as DWH with a non-distributed OLAP database engine | Parquet based DWH with non-distributed query engine set-up (e.g. SparkSQL or DuckDB) | Scalability and performance (e.g. SparkSQL >> DuckDB) |
+| Exploratory data science or ML use cases | Use the generated Parquet files which as _"near lossless"_ for enhanced data science workflows | Can either use the Parquet or custom schema to power dashboards or reports |
```

Comment:
We can probably delete this row; I am not sure if adds value.

Reply:
I did think it was worth making this explicit. Ok to remove it if you think its superfluous or obvious

Reply:
Okay sure.